index.js

```diff
@@ -53,7 +53,7 @@ Promise.resolve()
     if (argv.watch && !(argv.output || argv.replace || argv.dir)) {
       error('Cannot write to stdout in watch mode')
     }
-    if (input && input.length) return globber(input)
+    if (input && input.length) return globber(input, { dot: Boolean(argv.dot)})
```

Comment:
No need to wrap with `Boolean` here; yargs will take care of the casting itself.